templates/install_puppet.bat.erb

```diff
@@ -0,0 +1,11 @@
+:wait_for_pid
+FOR /F "tokens=*" %%A IN ('tasklist /FI "PID eq <%= @puppet_agent_pid %>"  /NH') DO set _task=%%A
+echo %_task% | findstr "No tasks are running" >nul
+IF NOT %errorlevel% == 0 ( GOTO wait_for_pid )
+
+
+msiexec.exe /qn /i <%= @_msi_location %> /l*v <%= @_logfile %>
```

Comment:
Do we want to wait this process or is it okay to let it roll?

I'd prefer `start /w`

And then check %ERRORLEVEL% right after to ensure it came out as 0 or 3010.